lib/app/app_widget.dart

```diff
@@ -40,54 +68,38 @@ class _AppWidgetState extends State<AppWidget> {
     return themeData;
   }
 
+  ThemeData get _darkThemeData {
+    final themeData = ThemeData(
+      brightness: Brightness.dark,
+      backgroundColor: Colors.red,
+      primaryColor: Colors.green,
+      // ignore: deprecated_member_use
+      accentColor: Colors.yellow,
+      textTheme: UiKit.text.textTheme,
+    );
+
+    return themeData;
+  }
+
   @override
   Widget build(BuildContext context) {
-    return BlocProvider<DeepLinkCubit>(
-      create: (context) => DeepLinkCubit(),
-      child: BlocProvider<QueryByExampleCubit>(
-        create: (context) => QueryByExampleCubit(),
-        child: BlocProvider<WalletBloc>(
-          create: (context) => WalletBloc(CredentialsRepository()),
-          child: BlocProvider<ScanBloc>(
-            create: (context) => ScanBloc(DioClient(Constants.checkIssuerServerUrl, Dio()), context.read<WalletBloc>()),
-            child: BlocProvider<QRCodeBloc>(
-              create: (context) => QRCodeBloc(
-                DioClient(Constants.checkIssuerServerUrl, Dio()),
-                context.read<ScanBloc>(),
-                context.read<QueryByExampleCubit>(),
-              ),
-              child: BlocProvider<ProfileBloc>(
-                create: (context) => ProfileBloc(),
-                child: BlocProvider<DIDBloc>(
-                  create: (context) => DIDBloc(),
-                  child: MaterialApp(
-                    title: 'Credible',
-                    routes: {
-                      '/splash': (context) => SplashPage(),
-                    },
-                    initialRoute: '/splash',
-                    theme: _themeData,
-                    localizationsDelegates: [
-                      AppLocalizations.delegate,
-                      GlobalMaterialLocalizations.delegate,
-                      GlobalWidgetsLocalizations.delegate,
-                      GlobalCupertinoLocalizations.delegate,
-                    ],
-                    supportedLocales: const <Locale>[
```

Comment:
We don't need to specify the supported locales anymore ?

Reply:
main.dart
  supportedLocales: AppLocalizations.supportedLocales,
  
  app_localisation.dart
  static const List<Locale> supportedLocales = <Locale>[
    Locale('de'),
    Locale('en'),
    Locale('es'),
    Locale('fr'),
    Locale('it')
  ];
  
  This includes all the supported languages.